### ISSUE TITLE
Accessibility: Textarea: Placeholder text within the Text area is not read, when assessed using the screen reader

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3438]
Bug

Description: Placeholder text within the Text area is not read, when assessed using the screen reader

Expected: Ideally, placeholder text should be read when text area is accessed.

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/94517834-675c4a80-0246-11eb-8796-c06a0760d4fe.png)

**Comment by kavya-b** [comment]
@puru-hk as verified with latest netlify, screen reader now reads the placeholder text. This was probably fixed as part of #4057 as well, hence closing this issue.
